algorithms/nepali/stem_Unicode.sbl

```diff
@@ -0,0 +1,55 @@
+/*
+ * Authors:
+ * - Ingroj Shrestha <ing.stha@gmail.com>, Nepali NLP Group
+ * - Oleg Bartunov <obartunov@gmail.com>, Postgres Professional Ltd.
```

Comment:
It seems you're submitting code other people wrote - have they both agreed to license this code under the 3 clause BSD licence (as in `COPYING`)?

Reply:
Yes, they agreed and they are aware about the pull request. And Oleg Bartunov is my boss.

Reply:
Hello, as a long time postgresql developer I am certainly  happy with BSD license ! The reason, we need the stemmer to be a part of snowball distribution, is that we want to submit default nepali full-text configuration (stemmer needed for that)  to PostgreSQL core and postgres community wants sort of approval  from snowball community.  We are a bit hurry, since the development stage is about to close in a couple of days.

Reply:
Ingroj Shrestha agreed with BSD license. He also asked to add Shreeya Singh Dhakal as author of the algorithm.